src/modules/scraper/entry-point/http/controller.ts

```diff
@@ -117,10 +118,20 @@ export class ScraperController {
     for (let depositId = fromDepositId; depositId <= toDepositId; depositId++) {
       await this.scraperQueuesService.publishMessage<DepositReferralQueueMessage>(ScraperQueue.DepositReferral, {
         depositId,
+        rectifyStickyReferralAddress: true,
       });
     }
   }
 
+  @Post("scraper/referral-address-reindex")
+  @Roles(Role.Admin)
+  @UseGuards(JwtAuthGuard, RolesGuard)
+  @ApiBearerAuth()
+  @ApiTags("scraper")
+  async submitReindexReferralAddressJob(@Body() body: SubmitReindexReferralAddressJobBody) {
```

Comment:
The intention of this job is just to allow manual reindexing in the case that somebody reports an issue?

Reply:
Yes. It takes ALL the deposits within a timeframe (`fromDate` -> `toDate`) and it will re-compute the referral addresses OR the sticky referral addresses by submitting the deposits ids to the `DepositReferralConsumer`